.github/workflows/deploy.yml

```
@@ -0,0 +1,153 @@
+      - name: Deploy services to EC2
+        run: |
+          ssh -o StrictHostKeyChecking=no ec2-user@${{ secrets.EC2_HOST }} << EOF
+          cd ~/teamo
+
+          # 디렉토리 구조 확인
+          mkdir -p ./nginx/certbot
+          mkdir -p ./nginx/www
+          
+          # 기존 컨테이너 중지
+          docker rm -f teamo certbot || true
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)